03 Writing Algorithms/24 Reality Modeling/08 Options Models/03 Exercise/05 Model Structure.html

```diff
@@ -1,10 +1,9 @@
-<p>Option exercise models should extend the <code>DefaultExerciseModel</code> class. Extensions of the <code>DefaultExerciseModel</code> class must implement the <code>OptionExercise</code> method, which receives <code>Option</code> and <code>OptionExerciseOrder</code> objects and then returns a list of <code>OrderEvent</code> objects that contain the order fill information.</p>
+<p>Option exercise models should implement the <code>IOptionExerciseModel</code> interface. The <code>IOptionExerciseModel</code> interface must implement the <code>OptionExercise</code> method, which receives <code>Option</code> and <code>OptionExerciseOrder</code> objects and then returns a list of <code>OrderEvent</code> objects that contain the order fill information.</p>
```

Comment:
This is tricky to explain because Python and C# are different. C# needs to implement `IOptionExerciseModel`, but Python doesn't. In Python, we just need to implement `OptionExerciseOrder`.

So, this part:
"Option exercise models should implement the <code>IOptionExerciseModel</code> interface." is only valid for C#.
The second part can use:
"The model must implement the <code>OptionExercise</code> method"
which will still be valid without mentioning the interface.